plugin/storage/es/options.go

```diff
@@ -29,45 +29,54 @@ import (
-	suffixPriorityServiceTemplate        = ".prioirity-service-template"
-	suffixPriorityDependenciesTemplate   = ".prioirity-dependencies-template"
-	suffixBulkSize                       = ".bulk.size"
-	suffixBulkWorkers                    = ".bulk.workers"
-	suffixBulkActions                    = ".bulk.actions"
-	suffixBulkFlushInterval              = ".bulk.flush-interval"
-	suffixTimeout                        = ".timeout"
-	suffixIndexPrefix                    = ".index-prefix"
-	suffixIndexDateSeparator             = ".index-date-separator"
-	suffixIndexRolloverFrequencySpans    = ".index-rollover-frequency-spans"
-	suffixIndexRolloverFrequencyServices = ".index-rollover-frequency-services"
-	suffixIndexRolloverFrequencySampling = ".index-rollover-frequency-adaptive-sampling"
-	suffixServiceCacheTTL                = ".service-cache-ttl"
-	suffixTagsAsFields                   = ".tags-as-fields"
-	suffixTagsAsFieldsAll                = suffixTagsAsFields + ".all"
-	suffixTagsAsFieldsInclude            = suffixTagsAsFields + ".include"
-	suffixTagsFile                       = suffixTagsAsFields + ".config-file"
-	suffixTagDeDotChar                   = suffixTagsAsFields + ".dot-replacement"
-	suffixReadAlias                      = ".use-aliases"
-	suffixUseILM                         = ".use-ilm"
-	suffixCreateIndexTemplate            = ".create-index-templates"
-	suffixEnabled                        = ".enabled"
-	suffixVersion                        = ".version"
-	suffixMaxDocCount                    = ".max-doc-count"
-	suffixLogLevel                       = ".log-level"
-	suffixSendGetBodyAs                  = ".send-get-body-as"
+	suffixUsername                           = ".username"
+	suffixPassword                           = ".password"
+	suffixSniffer                            = ".sniffer"
+	suffixSnifferTLSEnabled                  = ".sniffer-tls-enabled"
+	suffixTokenPath                          = ".token-file"
+	suffixPasswordPath                       = ".password-file"
+	suffixServerURLs                         = ".server-urls"
+	suffixRemoteReadClusters                 = ".remote-read-clusters"
+	suffixMaxSpanAge                         = ".max-span-age"
+	suffixAdaptiveSamplingLookback           = ".adaptive-sampling.lookback"
+	suffixNumShards                          = ".num-shards"
+	suffixNumReplicas                        = ".num-replicas"
+	suffixNumShardsSpan                      = ".num-shards-spans"
+	suffixNumShardsServices                  = ".num-shards-services"
+	suffixNumShardsSampling                  = ".num-shards-sampling"
+	suffixNumShardsDependencies              = ".num-shards-dependencies"
+	suffixNumReplicasSpan                    = ".num-replicas-spans"
+	suffixNumReplicasServices                = ".num-replicas-services"
+	suffixNumReplicasSampling                = ".num-replicas-sampling"
+	suffixNumReplicasDependencies            = ".num-replicas-dependencies"
```

Comment:
similar to above - can we have a single helper function that defines these? (including cmd/es-rollover/app/init/flags.go)